### PR TITLE
Show correct error messages on negative pids (#86)

### LINKED ISSUE #86: [BUG] incorrect pid validation in `proc` command
**Describe the bug**
`grofer proc -p [PID]` should give an error when an invalid PID is entered (negative values and some cases 0 as well as 0 might indicate the `sched` or the `swapper` process), if a PID equal to the `defaultProcPid` is entered, it runs the `grofer proc` command which shows info about all processes. 

**To Reproduce**
`grofer proc -p -1`

**Expected behavior**
The error shows for PID 0 and other negative numbers other than -1 should be shown for PID = -1 as well. 

**Desktop (please complete the following information):**
 - OS: Kubuntu 20.04
 - Go Version 1.14.7

**Additional context**
Looking at `cmd/proc.go` should help!

### PR DESCRIPTION
**Pull request by Samyak2**
# Description

Sets the default PID to 0, this ensures that the correct error message is shown when a negative PID is given. When PID 0 is given (`grofer proc -p 0`) it is considered to be the same as `grofer proc` i.e., all running processes are shown.

I'm not sure if this should be added to the README documentation, I can do this if needed.

WARNING: huge changes, might take a while to review :stuck_out_tongue_closed_eyes: 

Fixes #86 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] I have read the [contribution guidelines](https://github.com/pesos/grofer/blob/master/CONTRIBUTING.md) and followed it as far as possible. 
- [x] I have performed a self-review of my own code (if applicable)
- [x] I have commented my code, particularly in hard-to-understand areas (if applicable)
- [x] I have run `go fmt` on my code ([reference](https://blog.golang.org/gofmt))
- [ ] I have made corresponding changes to the documentation (if applicable)
- [x] My changes generate no new warnings 
- [x] Any dependent and pending changes have been merged and published

### CODE REVIEW

**Review by MadhavJivrajani — CHANGES_REQUESTED** [review]
Can you mention this behavior in the README as well as the help section of the `proc` command?

**Review by MadhavJivrajani — APPROVED** [review]
LGTM

**Comment by MadhavJivrajani** [comment]
Will wait for a review from @Gituser143 before merging.

**Review by Gituser143 — APPROVED** [review]

**Comment by Gituser143** [comment]
@Samyak2 @MadhavJivrajani After intense perusal of the highly complex changes (literally 2 lines) made here, I approve xD

**Comment by Gituser143** [comment]
@MadhavJivrajani Why is Travis still here?

**Comment by MadhavJivrajani** [comment]
> @MadhavJivrajani Why is Travis still here?

Exactly 
I've deleted the webhook as well. I'm not sure why builds are still getting triggered

**Pull request merged**
